doc/sphinx/language/gallina-extensions.rst

```diff
@@ -100,19 +100,22 @@ to be all present if the missing ones can be inferred or prompted for
        Rat_bottom_cond := O_S 1;
        Rat_irred_cond := one_two_irred |}.
 
-This syntax can be disabled globally for printing by
+This syntax can be disabled globally for display by
 
 .. cmd:: Unset Printing Records
 
-For a given type, one can override this using either
+For a given type, one can override this for specified types by adding entries to one of two tables.  Use
 
-.. cmd:: Add Printing Record @ident
+.. table:: Printing Record @qualid
+   :name: Printing Record
 
-to get record syntax or
+to see record syntax or
 
-.. cmd:: Add Printing Constructor @ident
+.. table:: Printing Constructor @qualid
+   :name: Printing Constructor
 
-to get constructor syntax.
+to see constructor syntax.  See the :cmd:`Add @table` and :cmd:`Remove @table` commands
+to change the set of overridden types.
```

Comment:
While you are at it, can you reformulate this paragraph so that each object (flag, table) gets a body (indented description).

Reply:
Done